packages/components/src/components/send-email-template/send-email-template.jsx

```diff
@@ -45,7 +44,7 @@ const SendEmailTemplate = ({
         setIsResendBtnDisabled(true);
         resend_interval = setInterval(() => {
             if (--timeout) {
-                setResendEmailBtnText(txt_resend_in.replace('{{seconds}}', timeout));
+                setResendEmailBtnText(`Resend email in ${timeout}s`);
```

Comment:
Should be localized.

```suggestion
                setResendEmailBtnText(localize('Resend email in {{ timeout_in_seconds }}s', {  timeout_in_seconds: timeout }));
```